test/built-ins/ShadowRealm/prototype/importValue/throws-if-import-value-does-not-exist.js

```diff
@@ -7,31 +7,20 @@ description: >
 info: |
   RealmImportValue ( specifierString, exportNameString, callerRealm, evalRealm, evalContext )
 
-    Assert: Type(specifierString) is String.
-    Assert: Type(exportNameString) is String.
-    Assert: callerRealm is a ShadowRealm Record.
-    Assert: evalRealm is a ShadowRealm Record.
-    Assert: evalContext is an execution context associated to a ShadowRealm instance's [[ExecutionContext]].
-    Let innerCapability be ! NewPromiseCapability(%Promise%).
-    Let runningContext be the running execution context.
-    If runningContext is not already suspended, suspend runningContext.
-    Push evalContext onto the execution context stack; evalContext is now the running execution context.
-    Perform ! HostImportModuleDynamically(null, specifierString, innerCapability).
-    Suspend evalContext and remove it from the execution context stack.
-    Resume the context that is now on the top of the execution context stack as the running
-    execution context.
-    Let steps be the steps of an ExportGetter function as described below.
+    ...
+    13. Let steps be the steps of an ExportGetter function as described below.
+    ...
 
   An ExportGetter function is an anonymous built-in function with a [[ExportNameString]]
   internal slot. When an ExportGetter function is called with argument exports,
   it performs the following steps:
 
-    Assert: exports is a module namespace exotic object.
-    Let f be the active function object.
-    Let string be f.[[ExportNameString]].
-    Assert: Type(string) is String.
-    Let hasOwn be ? HasOwnProperty(exports, string).
-    If hasOwn is false, throw a TypeError exception.
+    1. Assert: exports is a module namespace exotic object.
+    2. Let f be the active function object.
+    3. Let string be f.[[ExportNameString]].
+    4. Assert: Type(string) is String.
+    5. Let hasOwn be ? HasOwnProperty(exports, string).
+    6. If hasOwn is false, throw a TypeError exception.
```

Comment:
Nit pick: AFAICT @rwaldron is avoiding the step numbers as they frequently change in the specs so either way is fine and correct enough for this frontmatter.

Reply:
Yeah, it's "whatever". I leave them out now because I'm sick of updating the numbers when one step changes. Easier to drop in the change and move on.

Reply:
Hmm, interesting point. I was thinking that having the step numbers may help to distinguish that these lines are spec steps rather than descriptive lines. What do you think we just number them all with `1.` just like what we have in the spec text?

Reply:
I'm neutral to any of these options. the 1. might help indicating it's an ordered list

Reply:
@legendecas honestly, you are more than welcome to include the line numbers if you'd like <3